src/actions/Listing.js

```diff
@@ -8,11 +8,42 @@ export const ListingConstants = keyMirror(
   {
     FETCH_IDS: null,
     FETCH_IDS_SUCCESS: null,
-    FETCH_IDS_ERROR: null
+    FETCH_IDS_ERROR: null,
+    FETCH_FEATURED_HIDDEN: null
   },
   'LISTING'
 )
 
+export function fetchFeaturedHiddenListings(networkId) {
+  const readListingsFromGist = async (gitstUrl) => {
+    const response = await fetch(gitstUrl)
+    return (await response.text())
+      .split(',')
+      .map(listing => listing.trim())
+  }
+
+  return async function(dispatch) {
+    try{
+      if (process.env.FEATURED_LISTINGS_GIST === undefined || process.env.HIDDEN_LISTINGS_GIST === undefined){
+        console.error('Add environmental variables "FEATURED_LISTINGS_GIST" & "HIDDEN_LISTINGS_GIST" to enable hidden/featured listings functionality')
+        return
+      }
+
+
+      // the ?cache= part is for cache bust (so that previous results are not kept in cache)
+      const featuredListings = await readListingsFromGist(`https://raw.githubusercontent.com/OriginProtocol/origin-dapp/hidefeature_list/hidelist_${networkId}.json`)
```

Comment:
I'd say to remove reverences to "Gist" as that's a (now untrue) implementation detail. 
- `readListingsFromGist ()` --> `readListingsFromUrl()`
- `gitstUrl` --> `listingsUrl`
etc